charts/4allportal/values.yaml

```diff
@@ -281,6 +281,18 @@ webdav:
 
   podDisruptionBudget: true
 
+webdavEvent:
+  enabled: false
+  image:
+    registry: quay.io
+    repository: curl/curl
+    tag: latest
```

Comment:
@tasches @jpkraemer-mg 

Should we move it to `webdav.events.enabled` and `webdav.events.image`?

Reply:
It'd both look smoother and allow for more changeability later on, should there be any necessity of changing the webdav values.

Reply:
Ready